app/models/school_year.rb

```diff
@@ -0,0 +1,3 @@
+class SchoolYear < ApplicationRecord
```

Comment:
Since the school year doesn't actually (and shouldn't) belong to a school, can we call it an AcademicYear instead?